demos/string_output/README.md

```diff
@@ -0,0 +1,42 @@
+# String output model demo {#ovms_string_output_model_demo}
+## Overview
+
+This demo demonstrates usage of OVMS with model that returns string as an output.
+
+### Download Image Net modl
+
+```bash
+python3 download_model.py
+```
+
+### Start the OVMS container:
+```bash
+docker run -d -u $(id -u):$(id -g) -v $(pwd):/workspace -p 9178:9178 openvino/model_server:latest \
+--model_path /model --model_name text --rest_port 9178
+```
+
+## Send request
```

Comment:
Note which format it is. Also suggest using other available formats gRPC/REST KServe/TFS